internal/command/local/override/command.go

```diff
@@ -0,0 +1,267 @@
+		newDelete(cfg.LocalOverride),
+		newList(cfg.LocalOverride),
+	)
+
+	return cmd
+}
+
+func runOverride(out io.Writer, cfg *config.LocalOverrideCreate) error {
+	yellow := color.New(color.FgHiMagenta).SprintFunc()
+
+	if err := cfg.InitLocalConfig(); err != nil {
+		return err
+	}
+
+	if err := cfg.Validate(); err != nil {
+		return err
+	}
+
+	// Initialize API client
+	if err := cfg.API.InitAPIConfig(); err != nil {
+		return err
+	}
+
+	sandbox, err := getSandbox(cfg)
+	if err != nil {
+		return err
+	}
+
+	workloadNames, err := getOverrideWorkloadNames(sandbox, cfg.Workloads)
+	if err != nil {
+		return err
+	}
+
+	_, err = sbmgr.ValidateSandboxManager(sandbox.Spec.Cluster)
+	if err != nil {
+		return err
+	}
+
+	sandbox, overrideName, err := createSandboxWithMiddleware(cfg, sandbox, workloadNames)
+	if err != nil {
+		return err
+	}
+
+	sandbox, err = utils.WaitForSandboxReady(cfg.API, out, cfg.Sandbox, cfg.WaitTimeout)
+	if err != nil {
+		return err
+	}
+
+	err = waitForLocalReady(out, cfg)
+	if err != nil {
+		return err
+	}
+
+	if cfg.Detach {
+		if len(workloadNames) == 1 {
+			fmt.Fprintf(out, "Overriding traffic from sandbox '%s' workload '%s' to %s\n", cfg.Sandbox, workloadNames[0], cfg.To)
+		} else {
+			fmt.Fprintf(out, "Overriding traffic from sandbox '%s' workloads '%s' to %s\n", cfg.Sandbox, strings.Join(workloadNames, ", "), cfg.To)
+		}
+
+		fmt.Fprintf(out, "Traffic override will persist after this session ends\n")
+
+		helperMessage := fmt.Sprintf("%s local override delete %s --sandbox=%s", os.Args[0], overrideName, cfg.Sandbox)
+		fmt.Fprintf(out, "To remove override, run:\n\t%s\n", yellow(helperMessage))
+
+		return nil
+	}
+
+	// Set up signal handling for graceful shutdown
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	// Channel to listen for interrupt signal
+	sigChan := make(chan os.Signal, 1)
+	signal.Notify(sigChan, syscall.SIGINT, syscall.SIGTERM)
+
+	// Wait for signal or context cancellation
+	select {
+	case <-sigChan:
+		fmt.Fprintf(out, "\nSession terminated\n")
+		printOverrideProgress(out, fmt.Sprintf("Removing redirect in %s", cfg.Sandbox))
+		if err := deleteSandboxWithMiddleware(cfg, sandbox, overrideName); err != nil {
+			return err
+		}
+	case <-ctx.Done():
+		// Context was cancelled
+	}
+
+	return nil
+}
+
+func getSandbox(cfg *config.LocalOverrideCreate) (*models.Sandbox, error) {
+	sandboxParams := sandboxes.NewGetSandboxParams().WithOrgName(cfg.Org).WithSandboxName(cfg.Sandbox)
+
+	resp, err := cfg.Client.Sandboxes.
+		GetSandbox(sandboxParams, nil)
+	if err != nil {
+		return nil, err
+	}
+	return resp.Payload, nil
+}
+
+// getOverrideWorkloadNames returns the workload names for the given target workload. If no target workload is provided, all workload names are returned.
+// If a target workload is provided, but not found, an error is returned.
+func getOverrideWorkloadNames(sandbox *models.Sandbox, targetWorkloads []string) ([]string, error) {
```

Comment:
This function only works for forks, but it might be a local or virtual.

I would suggest:
1. only matching 1 workload
2. defaulting to the first workload, ordered by (virtual, fork, local)
3. allowing the user to specify the workload by name if the default is not desired.